localstack/services/transcribe/provider.py

```diff
@@ -85,6 +85,9 @@
 
 
 class TranscribeProvider(TranscribeApi, ServiceLifecycleHook):
+    def _get_bucket_and_key_from_s3_uri(self, s3_path: str) -> str:
```

Comment:
```suggestion
    def _get_bucket_and_key_from_s3_uri(s3_path: str) -> Tuple[str, Optional[str], Optional[str]]:
```
`partition` returns a 3-tuple. 

Also docstrings.

I would also move it to `localstack.services.s3.utils`

Reply:
Sorry, my bad, missed it, updating it.